openrndr-core/src/main/kotlin/org/openrndr/shape/RamerDouglasPeucker.kt

```diff
@@ -8,8 +8,7 @@ import org.openrndr.math.Vector2
  *
  * @author Edwin Jakobs
  *
- * @param points
- * @param epsilon when epsilon is less than the max distance between 2 points, recursively simplify
+ * @param epsilon When epsilon is less than the max distance between 2 points, recursively simplify.
```

Comment:
This description should be at method level not parameter. Is epsilon a distance value?